### PR TITLE
edit trip function works

### PR DESCRIPTION
**Pull request by JasmienVC**
image

Functionality works, frontend looks shit (we need js, I can fix it tomorrow :))

### CODE REVIEW

File: app/controllers/trips_controller.rb [hunk 1]
```diff
@@ -16,9 +16,18 @@ def create
   end
 
   def edit
+    @trip = Trip.find(params[:id])
   end
 
   def update
+    @trip = Trip.find(params[:id])
+    @profile.user = current_user
```
Comment (rubenvoss) [inline app/controllers/trips_controller.rb:24]: why are you setting the user of the profile here?

File: app/views/profiles/show.html.erb [hunk 2]
```diff
@@ -20,6 +20,10 @@
           <p><em><%= trip.crag.name %></em></p>
           <p><%= trip.start_date %> ⇨ <%= trip.end_date %></p>
         </div>
+         <% if @profile.user == current_user %>
```
Comment (rubenvoss) [inline app/views/profiles/show.html.erb:23]: this is always the case, right?

**Review by rubenvoss — CHANGES_REQUESTED** [review]

File: app/views/profiles/show.html.erb (hunk 2 above)
Comment (JasmienVC) [reply app/views/profiles/show.html.erb:23]: No, it's not. If I visit your profile, I am the current user, but @profile.user is you :)

File: app/controllers/trips_controller.rb (hunk 1 above)
Comment (JasmienVC) [reply app/controllers/trips_controller.rb:24]: yeah here you're right, we don't need it I think

**Review by rubenvoss — APPROVED** [review]

**Pull request merged**